label_studio/data_export/serializers.py

```diff
@@ -112,4 +123,19 @@ class Meta(ExportSerializer.Meta):
     serialization_options = SerializationOptionsSerializer(required=False, default=None)
 
 
+class ExportParamSerializer(serializers.Serializer):
+    interpolate_key_frames = serializers.BooleanField(default=settings.INTERPOLATE_KEY_FRAMES,
+                                                      help_text='Interpolate video key frames.',
+                                                      required=False)
+    download_resources = serializers.BooleanField(default=settings.CONVERTER_DOWNLOAD_RESOURCES,
+                                                  help_text='Download resources in converter.',
+                                                  required=False)
+    export_type = serializers.CharField(default='JSON',
```

Comment:
Is it ok, that we don't support the `exportType` parameter now?

Reply:
Fixed